### PR TITLE
Style check

### PR DESCRIPTION
**Pull request by odcinek**

### CODE REVIEW

File: lib/between_meals/changes/cookbook.rb [hunk 1]
```diff
@@ -54,10 +54,10 @@ def initialize(files, cookbook_dirs)
         # otherwise it was modified
         #   and will be re-uploaded
         if files.
-          select { |x| x[:status] == :deleted }.
-          map { |x| x[:path].match(%{.*metadata\.rb$}) }.
-          compact.
-          any?
+           select { |x| x[:status] == :deleted }.
```
Comment (jaymzh) [inline lib/between_meals/changes/cookbook.rb]: this should be... at elast one more, it's a continuation of the files...

File: lib/between_meals/changes/cookbook.rb [hunk 2]
```diff
@@ -78,7 +78,7 @@ def self.find(list, cookbook_dirs, logger)
           g = self.explode_path(x[:path], cookbook_dirs)
           g[:cookbook_dir] + '/' + g[:name] if g
         end.
-        map do |_, change|
+          map do |_, change|
```
Comment (jaymzh) [inline lib/between_meals/changes/cookbook.rb]: oh jesus.... we're chaining on an `if`, lol, we're evil.

File: lib/between_meals/repo.rb [hunk 3]
```diff
@@ -99,10 +99,6 @@ def files
       fail 'Not implemented'
     end
 
-    def latest_revision
```
Comment (jaymzh) [inline lib/between_meals/repo.rb]: why?

File: lib/between_meals/repo.rb [hunk 4]
```diff
@@ -111,10 +107,6 @@ def checkout
       fail 'Not implemented'
     end
 
-    def update
```
Comment (jaymzh) [inline lib/between_meals/repo.rb]: why?

File: lib/between_meals/repo/hg.rb [hunk 5]
```diff
@@ -120,14 +120,14 @@ def _username
 
       def check_refs(start_ref, end_ref)
         s = Mixlib::ShellOut.new(
-              "#{@bin} log -r #{start_ref}",
-              :cwd => @repo_path
+          "#{@bin} log -r #{start_ref}",
+          :cwd => @repo_path
             ).run_command
```
Comment (jaymzh) [inline lib/between_meals/repo/hg.rb]: the above to lines changing were correct, but now this one is no longre correct. It needs to come back... 4 spaces.

File: lib/between_meals/repo/hg.rb [hunk 6]
```diff
@@ -120,14 +120,14 @@ def _username
...
         s.error!
         if end_ref
           s = Mixlib::ShellOut.new(
-                "#{@bin} log -r #{end_ref}",
-                :cwd => @repo_path
+            "#{@bin} log -r #{end_ref}",
+            :cwd => @repo_path
               ).run_command
```
Comment (jaymzh) [inline lib/between_meals/repo/hg.rb]: same

File: lib/between_meals/repo/svn.rb [hunk 7]
```diff
@@ -116,14 +116,14 @@ def first_revision
 
       def check_refs(start_ref, end_ref)
         s = Mixlib::ShellOut.new(
-              "#{@bin} info -r #{start_ref}",
-              :cwd => @repo_path
+          "#{@bin} info -r #{start_ref}",
+          :cwd => @repo_path
             ).run_command
```
Comment (jaymzh) [inline lib/between_meals/repo/svn.rb]: same

File: lib/between_meals/repo/svn.rb [hunk 8]
```diff
@@ -116,14 +116,14 @@ def first_revision
...
         s.error!
         if end_ref
           s = Mixlib::ShellOut.new(
-                "#{@bin} info -r #{end_ref}",
-                :cwd => @repo_path
+            "#{@bin} info -r #{end_ref}",
+            :cwd => @repo_path
               ).run_command
```
Comment (jaymzh) [inline lib/between_meals/repo/svn.rb]: same

**Comment by jaymzh** [comment]
:+1: super stoked

File: lib/between_meals/repo.rb (hunk 3 above)
Comment (odcinek) [inline lib/between_meals/repo.rb]: duplicate

File: lib/between_meals/repo.rb (hunk 4 above)
Comment (odcinek) [inline lib/between_meals/repo.rb]: duplicate duplicate

File: lib/between_meals/changes/cookbook.rb (hunk 2 above)
Comment (odcinek) [inline lib/between_meals/changes/cookbook.rb]: I'm not gonna git blame this, just no

**Comment by jaymzh** [comment]
Ping?

**Comment by jaymzh** [comment]
Ping. ping. ping. ping. ping. ping. ping. ping.

Please?

This PR makes me happy. Can we rebase and fix up the 3 or 4 places the autofixer got confused? The new autocloser is going to close tihs PR if we don't.

**Comment by odcinek** [comment]
Updated rules and rubocop full run in https://github.com/facebook/between-meals/pull/27, essentially the same thing.

**Pull request closed without merging**